src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Core/PluginIdentity.cs

```diff
@@ -84,8 +84,8 @@ public static bool Equals(PluginIdentity a, PluginIdentity b)
         public override int GetHashCode()
         {
             return HashCodeUtils.CombineHashCodeValues(
-               this.Id.GetHashCode(),
-               this.Version.GetHashCode());
+               this.Id?.GetHashCode() ?? 0,
```

Comment:
Are these actually allowed to be null? Or is this just because the auto fixture isn't actually passing values?

Reply:
Removed as are not allowed to be null. I added guards in the other file.